docs/README.md

```diff
@@ -63,16 +63,16 @@ Note: when searching in JavaDoc, if your access is denied, please try removing t
 ## Extensions and utilities
 
 - [Android support](../android/README.md)
-- [Audio support](../extensions/audio/README.md)
```

Comment:
Why we need to rename extensions folder?

Reply:
because the [typesafe project accessor](https://docs.gradle.org/current/userguide/declaring_dependencies.html#sec:type-safe-project-accessors) clashes with a getter called `extensions`, so we either rename that or we disable that

Reply:
Let's disable this feature for now. Since this project is experimental, we can create a ticket to gradle and see if they can adda flag to change "extension" folder name